src/components/LoanFinding/PartnerSpotlightSection.vue

```diff
@@ -38,32 +47,61 @@ export default {
 		enableLoanCardExp: {
 			type: Boolean,
 			default: false
-		}
-	},
-	data() {
-		return {
-			mfiRecommendationsLoans: [],
-		};
+		},
+		spotlightData: {
+			type: Object,
+			default: () => {}
+		},
+		loans: {
+			type: Array,
+			default: () => []
+		},
 	},
 	computed: {
 		selectedChannelLoanIds() {
 			return this.mfiRecommendationsLoans?.map(element => element.id) ?? [];
 		},
-	},
-	methods: {
-		fetchMFILoans() {
-			// Load mfi recommendations loans data
-			return this.apollo.query({
-				query: mfiRecommendationsLoans,
-			}).then(({ data }) => {
-				this.mfiRecommendationsLoans = data?.fundraisingLoans?.values ?? [];
-				const numberLoans = this.mfiRecommendationsLoans.length;
-				if (!numberLoans) this.$kvTrackEvent('event-tracking', 'update', 'mfi-no-featured-loan-available');
-			});
+		headline() {
+			return this.spotlightData?.headline ?? '';
 		},
-	},
-	mounted() {
-		this.fetchMFILoans();
+		subheadline() {
+			return this.spotlightData?.subheadline ?? '';
+		},
+		partnerImage() {
+			return this.spotlightData?.image ?? '';
+		},
+		partnerImageFooter() {
+			return this.spotlightData?.imageFooter ?? '';
+		},
+		partnerText() {
+			return this.spotlightData?.partnerText ?? '';
+		},
+		carouselTitle() {
+			return this.spotlightData?.carouselTitle ?? '';
+		},
+		subheadsTitle() {
+			return this.spotlightData?.subheadsTitle ?? '';
+		},
+		subheads() {
+			return this.spotlightData?.subheads ?? '';
+		},
+		viewAllLink() {
+			return this.spotlightData?.viewAllLink ?? '';
+		}
 	},
 };
 </script>
+
+<style scoped>
```

Comment:
Some of these styles can be converted to TW with `@apply`

Reply:
Done